.github/workflows/stale.yml

```diff
@@ -3,8 +3,14 @@ on:
   schedule:
     - cron: "40 17 * * *"
 
+permissions:
+  contents: read
+
 jobs:
   stale:
+    permissions:
+      issues: write  # for actions/stale to close stale issues
+      pull-requests: write  # for actions/stale to close stale PRs
```

Comment:
The `actions/stale` action is using a custom secret, so we don't need to grant additional permissions in this case.

```suggestion
```